src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -720,27 +720,33 @@ def canDatasetGoToDisk(self, dataItem, isRelVal=False):
                 msg += "under campaign: {}. Letting it pass though...".format(dataItem['Campaign'])
                 self.logger.warning(msg)
                 return True
-            emailSubject = "[MSOutput] Campaign '{}' not found in central CouchDB".format(dataItem['Campaign'])
-            emailMsg = "Dataset: {} cannot have an output transfer rule ".format(dataItem['Dataset'])
-            emailMsg += "because its campaign: {} cannot be found in central CouchDB.".format(dataItem['Campaign'])
-            emailMsg += " In order to get output data placement working, add it ASAP please."
-            self.logger.critical(emailMsg)
+            # send alert via AlertManager API
+            alertName = "ms-output: Campaign not found: {}".format(dataItem['Campaign'])
+            alertSeverity = "high"
+            alertSummary = "[MSOutput] Campaign '{}' not found in central CouchDB".format(dataItem['Campaign'])
+            alertDescription = "Dataset: {} cannot have an output transfer rule ".format(dataItem['Dataset'])
+            alertDescription += "because its campaign: {} cannot be found in central CouchDB.".format(dataItem['Campaign'])
+            alertDescription += " In order to get output data placement working, add it ASAP please."
+            self.logger.critical(alertDescription)
             if self.msConfig["sendNotification"]:
-                self.emailAlert.send(emailSubject, emailMsg)
+                self.alertManagerAPI.sendAlert(alertName, alertSeverity, alertSummary, alertDescription, "ms-output")
```

Comment:
To avoid possible typos, I would suggest to define a new instance variable with the service name, something like:
```
self.alertServiceName = "ms-output"
```

which can also be used when defining the `alertName` a few lines above.